worldedit-core/src/main/java/com/fastasyncworldedit/core/math/heightmap/ArrayHeightMap.java

```diff
@@ -38,7 +38,7 @@ public void setSize(int size) {
     public double getHeight(int x, int z) {
         x = (int) Math.max(0, Math.min(width - 1, (x + size) * rx));
         z = (int) Math.max(0, Math.min(length - 1, (z + size) * rz));
-        return ((height[x][z] & 0xFF) * size) / 256d;
+        return ((height[x][z] & 0xFFFF) * size) / scale;
```

Comment:
the `& 0xFFFF` is not required anymore, as `char` is unsigned already.

Reply:
oop, true